fedbiomed/common/cli.py

```diff
@@ -256,6 +273,57 @@ def set_environ(self, environ):
         """Sets environ object"""
         self._environ = environ
 
+    @staticmethod
+    def config_action(this: 'CommonCLI', component: ComponentType):
```

Comment:
Maybe it would be nice to to try tune a little bit this `config_action` to avoid adding references to node/researcher within common ( `importlib.import_module("fedbiomed.node.environ").environ` ).

Maybe this might be done by calling:

```
        self._parser.add_argument(
            "--config",
            "-cf",
            nargs="?",
            action=self.config_action(self, ComponentType.NODE,
                                      importlib.import_module("fedbiomed.node.environ").environ),
            default="node_config.ini",
            help="Name of the config file that the CLI will be activated for. Default is 'node_config.ini'.")
```
```

Reply:
Hi @mvesin, 
Thanks for the idea. It is now implemented as we discussed.